src/components/Calculator.css

```diff
@@ -0,0 +1,36 @@
+button {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 18px;
+}
+
+.calc-container {
+  display: grid;
+  margin: 5vh 5vw;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: repeat(6, 5rem);
+}
```

Comment:
[ Optional Change ]

- [ ] It would be nice if you could add width to the container class as it spans the entire screen.  I suggest a 600px with will do.
```mdx
width: 600px;
```